pandas/tests/frame/test_arithmetic.py

```diff
@@ -100,6 +100,18 @@ def test_df_flex_cmp_constant_return_types_empty(self, opname):
 # Arithmetic
 
 class TestFrameFlexArithmetic(object):
+    def test_df_add_td64_columnwise(self):
+        # GH#22534 Check that column-wise addition broadcasts correctly
+        dti = pd.date_range('2016-01-01', periods=10)
+        tdi = pd.timedelta_range('1', periods=10)
+        tser = pd.Series(tdi)
+        df = pd.DataFrame({0: dti, 1: tdi})
+
+        result = df.add(tser, axis=0)
+        expected = pd.DataFrame({0: dti + tdi,
+                                 1: tdi + tdi})
```

Comment:
I generally prefer that the `expected` be as explicit as possible i.e., write out the actual result of those "sums" instead of the expression.  You can add a comment if needed to describe that we expected `dti + tdi` and `tdi + tdi`.

Reply:
I agree with you generally, but these types of tests its ok.